sphinx_gallery/tests/test_full.py

```diff
@@ -27,7 +27,7 @@
 
 N_FAILING = 2
 N_GOOD = N_TOT - N_FAILING
-N_RST = 16 + N_TOT  # includes module API pages, etc.
+N_RST = 16 + N_TOT  + 1 # includes module API pages, etc.
```

Comment:
sphinx_gallery/tests/test_full.py:30:24: E261 at least two spaces before inline comment
```suggestion
N_RST = 16 + N_TOT + 1  # includes module API pages, etc.
```